source/deployment-options/offline-installation.rst

```diff
@@ -494,10 +498,16 @@ Select your deployment type and follow the instructions to change the default pa
             INFO: The password for Wazuh API user wazuh-wui is +fLddaCiZePxh24*?jC0nyNmgMGCKE+2
             INFO: Updated wazuh-wui user password in wazuh dashboard. Remember to restart the service.
 
+      For the indexer's username and password, use the wazuh-keystore tool:
+
+         .. code-block:: console
+
+         # /var/ossec/bin/wazuh-keystore -f indexer -k username -v <INDEXER_USERNAME>
+         # /var/ossec/bin/wazuh-keystore -f indexer -k password -v <INDEXER_PASSWORD>
```

Comment:
Wrong format

```suggestion
      #. For the indexer's username and password, use the wazuh-keystore tool:

         .. code-block:: console

            # /var/ossec/bin/wazuh-keystore -f indexer -k username -v <INDEXER_USERNAME>
            # /var/ossec/bin/wazuh-keystore -f indexer -k password -v <INDEXER_PASSWORD>
```

Reply:
Done in [Make requested changes](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/9b2298127240e77d2df7ffd2b3e3f96e5a990cb1) commit

![image](https://github.com/wazuh/wazuh-documentation/assets/106568370/301c9114-7f88-4e44-a0f0-5ee3e49ab7ec)